src/SUMMARY.md

```diff
@@ -82,10 +82,10 @@
     - [SSL Ratios (public)](datasets/other/ssl/reference.md)
     - [Telemetry Aggregates](datasets/batch_view/telemetry_aggregates/reference.md)
     - [GLAM Aggregates](datasets/glam.md)
-  - [Experiment Datasets](tools/experiments.md)
+  - [Experiment Datasets](concepts/experiments.md)
```

Comment:
It feels like this document would actually better live in the section above ("data platform reference").

It's probably worth having a section here especially for experiments datasets, but we can make the table of contents a stub (like we do for derived datasets, above: https://docs.telemetry.mozilla.org/datasets/derived.html)